css/css-text/white-space/textarea-pre-wrap-010.html

```diff
@@ -1,11 +1,12 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
-<title>CSS Text level 3 Test: single preserved white space at the end of with white-space:pre-wrap overflow-wrap:break-spaces break-word in a textarea</title>
+<title>CSS Text level 3 Test: single preserved white space at the end of the line with white-space:break-spaces overflow-wrap:break-word in a textarea</title>
 <link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
 <link rel="help" href="https://drafts.csswg.org/css-text-3/#white-space-phase-2">
 <link rel="help" href="https://drafts.csswg.org/css-text-3/#overflow-wrap-property">
+<link rel="help" href="https://drafts.csswg.org/css-text-3/#white-space-property">
 <link rel="match" href="reference/textarea-pre-wrap-001-ref.html">
-<meta name="assert" content="a single preserved white space at the end of the line is wrapped when the white-space property is set to pre-wrap and overflow-wrap is break-spaces break-word in a textarea.">
+<meta name="assert" content="a single preserved white space at the end of the line is wrapped when the white-space property is set to break-spaces and overflow-wrap is break-word in a textarea.">
```

Comment:
What's the difference between this test and the one defined in overflow-wrap-break-word-003.html ?

Reply:
It's doing the same thing, but in a textarea instead of in a div. That is relevant because some browsers implement divs and textareas differently (because textarea is a replaced element). So, while both tests check the same statement in the spec, they are likely to catch different bugs.